### DISCUSSION TITLE
Show more button on profile ranks only loads 5 more scores

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#17867] [Q&A] [answered]
Clicking "Show more" in the "Ranks" section on a profile only loads 5 more scores, while the website loads 50. This makes it really tedious to see all your top plays (essentially having to click the button 20 times instead of 2).

https://user-images.githubusercontent.com/22381687/163814361-6238109b-3891-4dd4-9ad4-e517aedd9cfe.mp4

**Comment by frenzibyte** [comment, marked as answer]
Will be resolved in #17873.
